org.fooyin.fooyin.yaml

```diff
@@ -0,0 +1,106 @@
+app-id: org.fooyin.fooyin
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: fooyin
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '23.08'
+    add-ld-path: .
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--filesystem=/run/media'
+  - '--own-name=org.mpris.MediaPlayer2.fooyin'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--system-talk-name=org.freedesktop.UDisks2'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.gtk.Notifications'
```

Comment:
Are these two required? They don't go through the portal? 

Also why `org.gtk.Notifications`? It seems to be a Qt app.

Reply:
I don't see it having the ability to send any desktop notifications from a first look.

Reply:
Yes indeed, I thought this was needed for MPris notifications